src/js/program/background-media.ts

```diff
@@ -1,3 +1,9 @@
+import type Item from 'playlist/item';
```

Comment:
Really confused on how this is pulling a type from a `.js` file. When I tried this locally I ran into a bunch of errors so I resorted to this https://github.com/jwplayer/jwplayer/pull/3652/files#diff-9588e826a5bd96417540cd92b46bc3b6R25

https://github.com/jwplayer/jwplayer/pull/3652/files#diff-751861c9e54ddc75afa6f226a8b9ba2eR18

Your build passed which makes me really confused. Overall though I would say let's avoid `import type`ing non-tsed files and go with a generic placeholder for the time being

Reply:
ignore this comment because of https://github.com/jwplayer/jwplayer/pull/3652#discussion_r407578181

Reply:
Already doing this in model.ts